plugins/woocommerce-blocks/assets/js/blocks/product-filters/inner-blocks/status-filter/block.json

```diff
@@ -1,7 +1,7 @@
 {
-	"name": "woocommerce/product-filter-stock-status",
+	"name": "woocommerce/product-filter-status",
 	"version": "1.0.0",
-	"title": "Filter Options",
+	"title": "Status",
```

Comment:
```suggestion
	"title": "Status (Experimental)",
```

?

Reply:
Nice catch! Done in 95afca1c9192684567cfd4a2ba69eb25e71bd272